core/bind/core_bind.cpp

```diff
@@ -1265,6 +1270,8 @@ void _OS::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_power_seconds_left"), &_OS::get_power_seconds_left);
 	ClassDB::bind_method(D_METHOD("get_power_percent_left"), &_OS::get_power_percent_left);
 
+	ClassDB::bind_method(D_METHOD("request_permission", "name"), &_OS::request_permission);
```

Comment:
Could you add documentation for this method too since it's bound? Currently users might try to request all kinds of permissions using it, to find out that it doesn't work as they think.

Reply:
Alright :D